### ISSUE TITLE
Check the render method of `AlertModelize`

### ISSUE THREAD

**Issue by bunyaminmrcn** [WrathChaos/react-native-alert-modalize#1]
Hi . I created new project and tried the run given examle on expo and react-native
 but error happens like that. Both platforms gives same error. How can i solve this issue
**Note:** Wrapped with SafeAreaProvider

 **And added some additional dependencies**
 - `react-native-safe-area-context`
- `react-native-safe-area-view`

![ASDF](https://user-images.githubusercontent.com/30148504/71527619-43ea2500-28ed-11ea-8fcb-0e19e79bfe7f.png)

**Comment by kuraydev** [comment]
Hello @bunyaminmrcn,
This is not about the safe-area problem. Can you show me how to import the AlertModalize? Also, can you share with me the code example?

**Comment by bunyaminmrcn** [comment]
Gist link here. `SafeAreaView` imported from `react-native` and `react-native-safe-area-view` didn't work
[https://gist.github.com/bunyaminmrcn/9aff7f04bf8b68ac9abec611242a3546/564ef03c6b331cb4d0a2821c7c429ad0623cac75](url)

**Comment by kuraydev** [comment]
This gist link not working, can you check again? @bunyaminmrcn

**Comment by bunyaminmrcn** [comment]
https://gist.github.com/bunyaminmrcn/9aff7f04bf8b68ac9abec611242a3546
Why doesnt work link Its redirect to `issues`. It will propably work

**Comment by kuraydev** [comment]
It seems that it should work fine. Have you ever tried to run the library's example?

**Comment by bunyaminmrcn** [comment]
Yes , same error .Downgraded `react-native-modelize` package did not help

**Comment by kuraydev** [comment]
Can you share the full reproducible example repo? I need to investigate it because I could not reproduce it.

**Comment by bunyaminmrcn** [comment]
Here is URL. Thanks for being interested 

https://github.com/bunyaminmrcn/AlertModelize

**Comment by kuraydev** [comment]
@bunyaminmrcn Really sorry for the late response. Testing with your repo now.

**Comment by kuraydev** [comment]
@bunyaminmrcn When I cloned and `npm i` to your repo, it simply throws this error. Can you try with a new `react-native init AlertModalizeTest` and re-create the repo? 

![Screenshot 2019-12-30 at 12 06 43](https://user-images.githubusercontent.com/6813862/71575156-eaf0db80-2afc-11ea-9acb-1f6542446722.png)

**Comment by bunyaminmrcn** [comment]
@WrathChaos added missing script . If something wrong Please convert Line Endings character to your OS. 
  Not required script Just for issue :    https://github.com/facebook/metro/pull/468

**Comment by kuraydev** [comment]
@bunyaminmrcn I could not run this repo. That link did not solve the problem for me. Actually it should not even give me any error.
